api/models.py

```diff
@@ -4,6 +4,8 @@
 # Author: Guillaume Tucker <guillaume.tucker@collabora.com>
 # Author: Jeny Sadadia <jeny.sadadia@collabora.com>
 
+# pylint: disable=too-few-public-methods
```

Comment:
It might be good to add a comment about why it's OK to disable this check (basically, because models don't have methods...).